assets/css/style-mobile.css

```diff
@@ -444,6 +446,44 @@ h4 {
   margin: 20px 0;
 }
 
+.web-nav {
+  display: none;
+  background: rgba(96, 112, 255, 0.88);
+  height: 100%;
+  width: 100%;
+  top: 0;
+  margin-left: -15px;
+  position: fixed;
+}
+
+.web-nav img {
+  margin-left: 85%;
+  margin-top: 10%;
+}
```

Comment:
- [ ] Kindly reduce the size of the closing button image, it a little bigger than the one in the Figma design
### Your design :point_down: 
![Screenshot from 2023-06-21 21-49-13](https://github.com/kambereBr/Portfolio/assets/91142432/f8a68ad0-99f6-47f2-a898-bb7c747ff236)
### Figma design :point_down: 
![Screenshot from 2023-06-21 21-49-58](https://github.com/kambereBr/Portfolio/assets/91142432/4c339ec2-46f9-4415-be0f-0b66cebd4510)